test/llvm_tests/tmp.py

```diff
@@ -0,0 +1,16 @@
+from roadrunner import RoadRunner, Config
```

Comment:
The fact that this file is called 'tmp.py' makes me think it might not need to be in the source code?

Reply:
Yep, I've deleted it on my branch.